package.json

```diff
@@ -45,11 +39,14 @@
     "heroku-postbuild": "yarn build && yarn build-storybook",
     "serge": "yarn start-server",
     "snyk-protect": "set NODE_OPTIONS=–max_old_space_size=8192 snyk protect",
-    "prepare": "yarn run snyk-protect",
+    "prepare": "yarn run snyk-protect && husky install",
```

Comment:
In the dev instructions, I don't see a command for `yarn prepare`. Does it get triggered by some other process/call?

Reply:
Hi @IanMayo , this is a special life cycle scripts, more details at https://docs.npmjs.com/cli/v6/using-npm/scripts